### DISCUSSION TITLE
How do you do this with fyne

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3839] [Q&A] [answered]
###  Below is fyne's default Windows
<img width="550" alt="截屏2023-04-20 23 12 20" src="https://user-images.githubusercontent.com/1819457/233409761-8e1ffac9-6d0e-435a-b79a-a628aa77620d.png">

##  Whether the window title line is not displayed？ As follows
<img width="1002" alt="截屏2023-04-20 23 11 57" src="https://user-images.githubusercontent.com/1819457/233409741-9345fe01-e9a0-4be0-9164-5e9f698306f5.png">

**Comment by Jacalz** [comment, marked as answer]
If you are tall my about the split colours on the window title bar, then my best guess would be that it isn't possible. We are not using the native window toolkit but instead using GLFW and and drawing within the window. To the best of my knowledge, GLFW does not expose any such options (and it would not work cross-platform either way).

**Reply by junmaqiang** [reply]
think you asked me

**Reply by Jacalz** [reply]
Feel free to mark the comment above as answer if you are satisfied.
